articles/active-directory/develop/includes/remind-not-to-relay-token-nonaud.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Don't send your middle-tier OBO token to any non-audience party
+description: Include file warning that access tokens acquired for the middle-tier shouldn't be sent to any party except that which is identified by the audience claim.
+services: active-directory
+author: brianmel
+manager: CelesteDG
+
+ms.service: active-directory
+ms.subservice: develop
+ms.workload: identity
+ms.topic: include
+ms.date: 12/7/2021
+ms.author: brianmel
+ms.reviewer: brianmel
+ms.custom: aaddev
+---
+
+> [!WARNING]
+Access tokens issued to the middle-tier are intended for *use _only_* by the middle-tier and shouldn't be sent to any party other than the resource audience for which it was acquired. Relaying tokens from middle-tier to client is a discouraged practice due to increased risk of token interception over compromised SSL/TLS channels, inability to satisfy token binding and Conditional Access scenarios requiring claim step-up (for example, MFA, Sign-in Frequency), and incompatibility with admin-configured device-based policies (for example, MDM, location-based policies).
```

Comment:
@iambmelt Can you open this with "DO NOT <bad practice>" to be more direct?

This is a long warning, so you might want to really call out the badness. How about something like this:

```suggestion
**DO NOT** send access tokens that were issued to the middle tier to any other party. Access tokens issued to the middle tier are intended for use _only_ by that middle tier.

Security risks of relaying access tokens from a middle-tier resource to a client instead of the client getting the access tokens themselves include:

- Increased risk of token interception over compromised SSL/TLS channels
- Inability to satisfy token binding and Conditional Access scenarios requiring claim step-up (for example, MFA, Sign-in Frequency).
- Incompatibility with admin-configured device-based policies (for example, MDM, location-based policies).
```

Renders like so. Note I'm saying "middle-tier resource" in a couple spots - feel free to correct if it's not always a _resource_ [server].

> **Warning**
> **DO NOT** send access tokens that were issued to the middle tier to any other party. Access tokens issued to the middle tier are intended for use _only_ by that middle-tier resource.
>
> Security risks of relaying access tokens from a middle-tier resource to a client instead of the client getting the access tokens themselves include:
>
> - Increased risk of token interception over compromised SSL/TLS channels
> - Inability to satisfy token binding and Conditional Access scenarios requiring claim step-up (for example, MFA, Sign-in Frequency).
> - Incompatibility with admin-configured device-based policies (for example, MDM, location-based policies).

Reply:
LGTM ca51dc36aba211ed0ec353630adbfa05af7d6a8d